query/common/src/main/java/org/codice/ddf/admin/common/fields/common/CredentialsField.java

```diff
@@ -35,8 +35,6 @@
 
     public CredentialsField() {
         super(DEFAULT_FIELD_NAME, FIELD_TYPE_NAME, DESCRIPTION);
-        this.username = new StringField("username");
-        this.password = new StringField("password");
     }
 
     public CredentialsField username(String username) {
```

Comment:
Should this more specifically be a `UPCredentialsField`, leaving open the possibility of having other types of credential fields in the future?